libfwupdplugin/fu-dpaux-device.c

```diff
@@ -70,11 +70,25 @@ fu_dpaux_device_probe(FuDevice *device, GError **error)
 static gboolean
 fu_dpaux_device_setup(FuDevice *device, GError **error)
 {
+	FuContext *ctx = fu_device_get_context(device);
 	FuDpauxDevice *self = FU_DPAUX_DEVICE(device);
 	FuDpauxDevicePrivate *priv = GET_PRIVATE(self);
+	const gchar *name = fu_udev_device_get_sysfs_attr(FU_UDEV_DEVICE(self), "name", NULL);
 	guint8 buf[FU_STRUCT_DPAUX_DPCD_SIZE] = {0x0};
 	g_autoptr(GByteArray) st = NULL;
 
+	/* ignore all Framework FRANDGCP07 BIOS version 3.02 */
+	if (name != NULL && g_str_has_prefix(name, "AMDGPU DM") &&
+	    fu_context_has_hwid_guid(ctx, "bf106e31-bdab-5a7b-93b1-5d5b9cd376c0")) {
```

Comment:
@JohnAZoidberg specifically is this the right hwid to use to cover it?

I don't know if there are multiple production strings that need to be covered.

Something else on my mind is we can key off the uefi firmware version and uefi esrt guid instead of hwid.

Or it's possible to get the IMU firmware version, but the kernel doesn't export it from sysfs until newer kernels. Let me see if it's possible to read outside the kernel relatively easily.

Reply:
I think that looking at his HWIDs using `32d49d99-414b-55d5-813b-12aaf0335b58` is a better HWID based quirk.

Reply:
Ack, agreed. Fixed.